pkg/connector/ble/ble.go

```diff
@@ -36,6 +36,10 @@ var (
 	mu     sync.Mutex
 )
 
+func SetDebugLog() {
```

Comment:
This doesn't belong in `ble.go`, since the logging level is shared between sub-packages.

I think the solution here is to move `internal/log` to `pkg/log`.

A more involved solution would be to migrate to `slog` or other structured logging package.